code/modules/antagonists/wizard/equipment/soulstone.dm

```diff
@@ -62,7 +62,7 @@
 		return
 	if(!ishuman(M))//If target is not a human.
 		return ..()
-	if(iscultist(M))
+	if(iscultist(M)	&& iscultist(user))
 		to_chat(user, "<span class='cultlarge'>\"Come now, do not capture your bretheren's soul.\"</span>")
```

Comment:
```dm
if(iscultist(M))
    if(iscultist(user))
 		to_chat(user, "<span class='cultlarge'>\"Come now, do not capture your bretheren's soul.\"</span>")
    return
```

Reply:
Actually, just always return if !iscultist(user)

Reply:
It needs to have a message for both cultists and non, silently not working isn't exactly intuitive that the individual is a cultist.

Reply:
why did you request this changed?